geode-serialization/src/main/java/org/apache/geode/internal/serialization/KnownVersion.java

```diff
@@ -205,12 +205,12 @@
       new KnownVersion("GEODE", "1.13.0", (byte) 1, (byte) 13, (byte) 0, (byte) 0,
           GEODE_1_13_0_ORDINAL, true);
 
-  private static final short GEODE_1_13_1_ORDINAL = 121;
+  private static final short GEODE_1_13_2_ORDINAL = 121;
```

Comment:
If we are removing 1.13.1 we need to remove it from the unreleased 1.14 and develop branches also.

Reply:
I take that back, we can't replace this ordinal because we have already released versions with this ordinal defined. It may not be safe to have it suddenly mean something different. The safer thing to do is define a new ordinal 122 for 1.13.2.

Reply:
I'm not sure I follow your reasoning here. 1.13.1 released with ordinal 120. 1.13.2 released with ordinal 121.

The problem here is that the name of this constant is wrong. It should be called GEODE_1_13_2_ORDINAL, because that is the release in which this ordinal was used.

I guess there is a question of whether some external code is using this constant, since there will no longer be a constant named GEODE_1_13_1_ORDINAL. However this is an internal class, so it can be expected to change - in fact this whole class was renamed in 1.14. The most likely project to use these internal constants would be spring data geode, but that project does not use this internal class.

Reply:
If you are comfortable then I am comfortable.